### PR TITLE
[broker] add brokerServiceUrl in TopicStats

### PR DESCRIPTION
**Pull request by Jason918**
### Motivation

It's quite useful to get topic info with the cmd `bin/pulsar-admin topics stats <topic>` when doing some trouble shooting work.
By adding brokerServiceUrl in TopicStats, we can save some trouble running another lookup cmd.

### Modifications

Add brokerServiceUrl in class TopicStats. And assigned with the value `brokerService.pulsar().getBrokerServiceUrl()`

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API: (no)
  - The schema: (no)
  - The default values of configurations: (no)
  - The wire protocol: (no)
  - The rest endpoints: (no)
  - The admin cli options: (no)
  - Anything that affects deployment: (no)

### Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by Jason918** [comment]
Hi @jiazhai , @codelipenghui , can you please take a look?

**Comment by Anonymitaet** [comment]
@Jason918 thanks for your contribution. For this PR, do we need to update docs?

**Comment by Jason918** [comment]
> @Jason918 thanks for your contribution. For this PR, do we need to update docs?

Thx for the reminding. I've updated the docs in the new commit.

**Comment by Jason918** [comment]
@Anonymitaet The docs for topic stats cmd is not in sync with the implementation. Maybe we should open new PR to complete the docs.

File: site2/docs/admin-api-topics.md [hunk 1]
```diff
@@ -186,6 +186,8 @@ You can check the following statistics of a given non-partitioned topic.
   -   **averageMsgSize**: The average size (in bytes) of messages published within the last interval.
 
   -   **storageSize**: The sum of the ledgers' storage size for this topic. The space used to store the messages for the topic.
+  
+  -   **brokerServiceUrl**: The broker URL which is serving the given topic.
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:190]:
```suggestion
  -   **brokerServiceUrl**: The broker URL which serves a given topic.
```

**Review by Anonymitaet — COMMENTED** [review]
Thanks for adding the docs. I've left a comment, PTAL.

**Comment by Anonymitaet** [comment]
> @Anonymitaet The docs for topic stats cmd is not in sync with the implementation. Maybe we should open new PR to complete the docs.

could you please add the docs? Then you can ping me to review, thanks.

**Review by 315157973 — COMMENTED** [review]
Could you add a unit test to make sure it works?

**Comment by Jason918** [comment]
> > @Anonymitaet The docs for topic stats cmd is not in sync with the implementation. Maybe we should open new PR to complete the docs.
> 
> could you please add the docs? Then you can ping me to review, thanks.



> Could you add a unit test to make sure it works?

Added some simple tests in PartitionedTopicStatsTest.java and PartitionedTopicStatsTest.java

**Review by cckellogg — CHANGES_REQUESTED** [review]
I don't think the brokerServiceUrl should be part of a topic stats response or object. There are already ways to get his information.

**Comment by Jason918** [comment]
> I don't think the brokerServiceUrl should be part of a topic stats response or object. There are already ways to get this information.

The owner broker of the topic may change during  two separate cmds, right? 
And ... it's more efficient doing some trouble shooting work.

**Pull request closed without merging**